homework_11/task1.py

```diff
@@ -0,0 +1,21 @@
+def convert_temperature(value, type):
+    if type == "C":
+        return value * 9/5 + 32
+    elif type == "F":
+        return (value - 32) * 5/9
+
+def main():
+
+    print("Converting Celsius to Fahrenheit:")
+    print("25 ->", convert_temperature(25, "C"))
+    print("0 ->", convert_temperature(0, "C"))
+    print("-10 ->", convert_temperature(-10, "C"))
+    
+    print()
```

Comment:
ეს ზედმეტ ხაზად რომ არ წერო, თუ არ გინდა რომ მიებას, ქვემოთ მიუწერ ენთერს "\n": 
    print("\nConverting Fahrenheit to Celsius:")